peopleops/recruiting/HIRING_FLOW.md

```diff
@@ -0,0 +1,81 @@
+
+2) Use a timezone conversion system like [***Savvy Time***](https://savvytime.com/converter/pdt-to-ist), to schedule the meetings that should occur after the initial screening.
+
+3) Use any video-chat service, like [Appear.in](http://appear.in/countable) to schedule an interview with your best candidates.
+
+4) You can check for more tools at our [peopleops folder](https://drive.google.com/drive/folders/1HY7LPI_h5F5WxaTW5v-1_DMUXcJIjjh1)
+
+## Initial Screening
+
+1) Create an account at ***Upwork*** and click on the CTA "Post a Job" to create a post according to your needs. Make sure you include all relevant keywords in your new ad since it will improve your lead capturing.
+
+2) Once it's created, check it by clicking at the navbar: JOBS > MY JOBS. It will appear at "My Postings" section, where you can see how many proposals you received and how many people messaged you.
+
+3) After some time you'd receive your first leads if you've set your job post correctly.
+     - If not, delete it and create another, making sure you're entering relevant keywords to it. If you're unsure about the job requirements, ask your supervisor's assistance.
+
+4) At "REVIEW PROPOSALS" step, click on "Filters". Now you should ***shortlist***(click on the thumbs-up icon) the candidates that pass the following suggested minimum criteria:
+ - Earned amount > 10k
+ - Job success > 90%
+ - Hours billed: 100+
+- English level: conversational
+- Talent type: freelancers & agencies
+
+5) Now send a message to the selected ones, asking for a quick (15 min) video chat interview. Check what timezone he's in, and use a [***time zone conversion tool***](https://savvytime.com/converter/pdt-to-ist) to make sure it will match yours.
+
+6) It's important to remember scheduling the interview time in an Alarm app (any app that remembers you about it) since once you get 2+ interviews on the same day, you'll have a hard time trying to remember when exactly did you scheduled what meeting.  Be careful to not confusing timezones!
+
+## Scheduling
+
+- Please use [Calendly](https://calendly.com/) to schedule your meetings and avoid timezone issues.
+
+
+## Interviewing
+
+- Prepare yourself to the interview. Take a look at our standard [interview questions](https://docs.google.com/document/d/130KCrw5WGYTGgJYKBVec3iEXxKkNoEkm1urHzEWznMo/edit#heading=h.7h8atrac1axk) and pick the ones that you find relevant for the situation
+
+- Communicate your supervisor about the scheduled meetings, since he should participate as well.
+
+- On this step you and your selected candidate should join a video-chat meeting room, so you can have a quick conversation and get non-verbal cues about the lead personality traits, values, etc.
+
+ - [Here's a link to countable's video-chat room](http://appear.in/countable)
+
+
+
+## Tests
+
+- Testing candidates could be done in a variety of ways and it depends upon the position you're hiring for.
```

Comment:
This sentence doesn't really say anything, i think it could be deleted

Reply:
You're right